inc/zoo/swar/associative_iteration.h

```diff
@@ -483,6 +493,82 @@ constexpr auto exponentiation_OverflowUnsafe_SpecificBitCount(
     );
 }
 
+using S = SWAR<4, u16>;
+
+/** Transforms a number into a number into a binary tally.
+ * E.g. 0b0011 (3) -> 0b0111
```

Comment:
You seem to mean this converts a binary to an unary

Reply:
yes lol thank you.